pkg/bloomcompactor/chunkcompactor.go

```diff
@@ -92,7 +92,7 @@ func buildBloomFromSeries(seriesMeta seriesMeta, fpRate float64, tokenizer compa
 	}
 
 	// Tokenize data into n-grams
-	tokenizer.PopulateSeriesWithBloom(&bloomForChks, chunks)
+	_ = tokenizer.PopulateSeriesWithBloom(&bloomForChks, chunks)
```

Comment:
Should we log the error here?

Reply:
Wasn't sure on this.  Right now, the function doesn't return an error, so we'd just be logging the same error again here.  I think as the compactor gets fleshed out some more, it will become apparent where to add the error handling and logging. wdyt?

Reply:
let's keep it then